src/main/resources/db/changelog/changes/09-create-shopping-carts-table.yaml

```
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: admin
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                    foreignKeyName: fk_shopping_carts_users
+                    referencedTableName: users
+                    referencedColumnNames: id
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```

```suggestion
                    nullable: false
                    
```
Add empty line at the end of each changelog